dm/config/subtask.go

```diff
@@ -44,6 +44,12 @@ var (
 	defaultMaxAllowedPacket = 64 * 1024 * 1024 // 64MiB, equal to TiDB's default
 )
 
+// DBTimeoutConfig specify the DB timeout configuration
+type DBTimeoutConfig struct {
```

Comment:
can we move it into `DBConfig`?

Reply:
I think DBTimeoutConfig should not set by user, so I put it outside

Reply:
can use `toml:"-" json:"-" YAML:"-"`

Reply:
fix it